docs/pages/access-controls/device-trust/jamf-integration.mdx

```diff
@@ -0,0 +1,303 @@
+step above.
+
+Save the following file as `/var/lib/teleport.yaml` and edit as needed:
+
+```yaml
+version: v3
+teleport:
+  # Necessary to write devices back to Teleport.
+  proxy_server: teleport.example.com:443 # CHANGEME
+
+jamf_service:
+  enabled: true
+  name: jamf
+  api_endpoint: https://yourtenant.jamfcloud.com # CHANGEME
+  username: teleport                             # CHANGEME
+  password_file: /var/lib/teleport/jamf_password.txt
+
+auth_service:
+  enabled: false
+
+proxy_service:
+  enabled: false
+
+ssh_service:
+  enabled: false
+```
+
+Change the following settings, as appropriate:
+
+- teleport.proxy_server
+- jamf_service.api_endpoint
+- jamf_service.username
+
+Finally, write your Jamf password to the `/var/lib/teleport/jamf_password` file:
+
+```code
+$ nano /var/lib/teleport/jamf_password # or use your favorite editor
+
+# Only the OS user that runs `teleport` should have access to the password file.
+$ chmod 400 /var/lib/teleport/jamf_password
+$ chown teleport /var/lib/teleport/jamf_password
+```
+
+## Step 3/4. Create a join token
+
+Jamf service requires an MDM token to write devices to Teleport.
+Create the token as shown below:
+
+```code
+$ tctl tokens add --type=mdm
+The invite token: 07c0565aa280595d7645b2964c36ebd1
+This token will expire in 30 minutes.
+
+Use this token to add an MDM service to Teleport.
+
+> teleport start \
+   --token=07c0565aa280595d7645b2964c36ebd1 \
+   --ca-pin=sha256:4b32d9c54b2b3332019d5f0720b8f9a603de03ace07d308bcd743465eee1f200 \
+   --config=/path/to/teleport.yaml
+```
+
+## Step 4/4. Start Jamf service
+
+Using the token created above, start the service:
+
+```code
+$ teleport start --config=/var/lib/teleport.yaml \
+   --token=07c0565aa280595d7645b2964c36ebd1 \
+   --ca-pin=sha256:4b32d9c54b2b3332019d5f0720b8f9a603de03ace07d308bcd743465eee1f200
+```
+
+The initial sync should happen in a few minutes:
+
+```
+2023-06-21T17:26:40-03:00 INFO [JAMF:1]    Jamf service successfully started pid:25757.1 service/service.go:228
+2023-06-21T17:26:40-03:00 INFO [JAMF:1]    Starting sync CutTime:0001-01-01 00:00:00 +0000 UTC FilterRSQL: Mode:1 OnMissing:DELETE pid:25757.1 service/service.go:261
+2023-06-21T17:26:40-03:00 INFO [JAMF:1]    Device sync report, page #0 deletes:0 failures:0 pid:25757.1 upserts:1 service/service.go:666
+2023-06-21T17:26:40-03:00 INFO [JAMF:1]    Sync complete pid:25757.1 service/service.go:277
+```
+
+Using the default configuration, the service will sync devices from Jamf every
+few hours. Once a day a full inventory sync is performed, enumerating all
+devices from Jamf and reflecting any additions or removals on Teleport.
+
+After the initial sync happens, you may verify the synced devices using `tctl
+devices ls`:
+
+```code
+$ tctl devices ls
+Asset Tag    OS      Enroll Status Device ID
+------------ ------- ------------- ------------------------------------
+CXXXXXXXXX17 macOS   not enrolled  20ec6373-9e8e-46e0-8f1c-47ad6b06a768
```

Comment:
Suggest adding additional demo device IDs around L1752 of `config.json`, and updating this example to use those presets.

Reply:
I don't think it's worth adding this many example devices to config.json. We do have an ID for single-device examples, but this is the exception.

Reply:
As long as this doesn't spark any false positives about leaked credentials in the repo, fine by me.

Reply:
I don't expect an UUID would cause us problems, but let me know if it does.